README.md

```diff
@@ -1074,6 +1074,61 @@ fga query **expand** <relation> <object> --store-id=<store-id> [--model-id=<mode
 }
 ```
 
+##### List Users
+
+###### Command
+fga query **list-users** --object <object> --relation <relation> --user-filter <user-filter> [--contextual-tuple "<user> <relation> <object>"]* --store-id=<store-id> [--model-id=<model-id>]
+
+###### Parameters
+* `--store-id`: Specifies the store id
+* `--object`: Specifies the object to list users for
+* `--relation`: Specifies the relation to search on
+* `--user-filter`: Specifies the type or userset to filter with
+* `--model-id`: Specifies the model id to target (optional)
+* `--contextual-tuple`: Contextual tuples (optional) (can be multiple)
+* `--context`: Condition context (optional)
+
+###### Example
+`fga query list-users --store-id=01H0H015178Y2V4CX10C2KGHF4 --object document:roadmap --relation can_view --user-filter user`
+`fga query list-users --store-id=01H0H015178Y2V4CX10C2KGHF4 --object document:roadmap --relation can_view --user-filter user --contextual-tuple "user:anne can_view folder:product"`
+`fga query list-users --store-id=01H0H015178Y2V4CX10C2KGHF4 --object document:roadmap --relation can_view --user-filter group#member --context '{"ip_address":"127.0.0.1"}`
+
+###### Response
+```json5
+{
+    {
+      "excluded_users": [
+        {
+          "object": {
+            "type": "user",
+            "id": "beth"
+          }
+        }
+      ],
+      "users": [
```

Comment:
I think this output might be unrealistic due to the differing types so not sure if it could be improved or if it's fine for an illustrative purpose?

Reply:
Because it's the CLI readme, let's have a reproducible case, as in something they can copy and run and get the same response, but we can add the other filters and responses in a separate section